rtcrtpreceiver.go

```diff
@@ -127,4 +131,6 @@ func (r *RTCRtpReceiver) Receive(parameters RTCRtpReceiveParameters) chan bool {
 // Stop irreversibly stops the RTCRtpReceiver
 func (r *RTCRtpReceiver) Stop() {
```

Comment:
Can you add a `mu Sync.mutex` and `closed bool` member to `RTCRtpReceiver`

Then only allow people to call close once, and if it is already closed just early return. Calling `close()` twice on a channel causes a panic